tests/consequence_prediction/vep_mapping_pipeline/test_consequence_mapping.py

```diff
@@ -0,0 +1,81 @@
+from consequence_prediction.vep_mapping_pipeline.consequence_mapping import extract_consequences
+
+
+def test_extract_consequences():
```

Comment:
Just as a suggestion, if you add a docstring to the test, the testing library (I think both unittest and pytest support this) will be able to automatically use the first line of it as a short test description, for example to display it if it fails.

This also applies to some cases where currently there are top level comments, which could also be converted into docstrings for the same purpose.